vulture_os/toolkit/api_parser/aws_bucket/aws_bucket.py

```diff
@@ -137,7 +137,7 @@ def execute(self):
             for filename in self._fetch_files():
                 self.update_lock()
                 file_data = self._download_file(filename)
-                print(file_data)
+                print(file_data) # WTH HERE?
```

Comment:
```suggestion
                print(file_data)
```

Reply:
This has absolutely no reason to be here
- don't use print() -> use logger instead
- this data could be printed in logs, but in debug only

Reply:
Keeping it as is for now, should be reviewed in the next version